<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKWebSocketClientDeviceController</class>
 <widget class="QWidget" name="SAKWebSocketClientDeviceController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Flow control</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Server url</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="serverHostLineEdit">
     <property name="text">
      <string notr="true">ws://localhost:12345</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="sendingTypeComboBox"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Web socket client information</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Client info</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="clientInfoLineEdit">
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
